toolkits/nature/packages/nature-facet/js/enhanced-facet.js

```diff
@@ -0,0 +1,114 @@
+import {makeArray} from '@springernature/global-javascript/src/helpers';
+import {Expander} from '@springernature/global-expander/js/expander';
+
+/**
+ * Constants
+ */
+
+const DATA_COMPONENT = 'data-facet-expander';
+
+const selectors = {
+	DATA_COMPONENT: `[${DATA_COMPONENT}]`,
+	FACET: '[data-facet]'
+};
+
+const classNames = {
+	SELECTED: 'c-facet__selected',
+	ELLIPSIS: 'c-facet__ellipsis',
+	CLEAR: 'c-facet__clear-selection',
+	BUTTON_CONTAINER: 'c-facet__button-container',
+	SUBMIT: 'c-facet__submit'
+};
+
+const buttonName = {
```

Comment:
Maybe buttonCopy?

Reply:
would you be persuaded with `buttonText`? `buttonCopy` sounds a bit like the button is being or has been cloned

Reply:
I think `buttonText` is amazing!